tests/test_commit_message.py

```diff
@@ -85,15 +85,15 @@ def test_commit_message_blocking_reviewers_requested(reviewer_text):
         "r?#group1",
         "r?#group1, #group2",
         "r?reviewer_one,#group1",
-        "r?#group1 r?reviewer_two",
+        "r?#group1 r?reviewer.two",
         "r?#group1! r?group2",
         "r?#group1 r?group2!",
         "r?#group1! r?group2!",
-        "r?#group1, reviewer_two!",
+        "r?#group1, reviewer.two!",
```

Comment:
I'd also like to see a test to ensure trailing periods aren't matched.

Reply:
Noted.

Reply:
`format_commit_message` actually uses the RE differently from other repos when it comes to parsing reviewers  -- it's doing the reverse, i.e. creating a reviewer list from a predetermined list of reviewers fetched via the Phabricator client, and thus the behaviour here is quite different. What it does do though is look for the first occurrence of `r=` using the RE and go from there. The behaviour here is actually more desirable, however this is broken in other repos.